src/GameView.cpp

```diff
@@ -84,8 +150,35 @@ void GameView::render(){
     this->App->clear();
     this->App->draw(levelSprite);
     this->App->draw(sprite_player);
+    this->App->draw(sprite_player);
+    if(levelManager.currentLevelName == "BEDROOM"){
```

Comment:
i've been thinking through this and we should have a list of sprites instead that gets rendere. This is going to be a bit complicated so i'll think through this more.